File: 2020/1Semestre/POO/projetoInterface/projetoInterface/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace projetoInterface
{
    class Pessoa
    {
        string nome, endereco;
        int idade;
        public Pessoa(string nome, string endereco, int idade)
        {
            this.nome = nome;
            this.endereco = endereco;
            this.idade = idade;
        }
        public string getNome()
        {
            return nome;
        }
        public string getEndereco()
        {
            return endereco;
        }
        public int getIdade()
        {
            return idade;
        }
    }
}

File: 2021/2Semestre/Tecnicas_programação_II/Aula-14-08/src/Pessoa.java
import java.io.Serializable;

public class Pessoa implements Serializable {
    private String nome, nacionalidade, dataNascimento;
    private String sexo;
    private int idade;

    public Pessoa() {
        //vazio
    }

    public Pessoa(String nome, String nacionalidade, String dataNascimento, String sexo, int idade ) {
        setNome(nome);
        setNacionalidade(nacionalidade);
        setSexo(sexo);
        setIdade(idade);
        setDataNascimento(dataNascimento);
    }

    public String getDataNascimento() {
        return dataNascimento;
    }
    public int getIdade() {
        return idade;
    }
    public String getNacionalidade() {
        return nacionalidade;
    }
    public String getNome() {
        return nome;
    }
    public String getSexo() {
        return sexo;
    }
    public void setDataNascimento(String dataNascimento) {
        this.dataNascimento = dataNascimento;
    }
    public void setIdade(int idade) {
        this.idade = idade;
    }
    public void setNacionalidade(String nacionalidade) {
        this.nacionalidade = nacionalidade;
    }
    public void setNome(String nome) {
        this.nome = nome;
    }
    public void setSexo(String sexo) {
        this.sexo = sexo;
    }

    public String ToSring(){
        return getNome()+';'+getSexo()+';'+getIdade()+';'+getDataNascimento()+';'+getNacionalidade();
    }
}
